src/graph.py

```diff
@@ -752,6 +741,105 @@ def perform_GO_enrichment_on_clusters(nxgraph, clustering):
     """
     pass
 
+
+###############################################################################
+            #######            Results           #######
+###############################################################################
+
+def summarize_grid_search(results, mode, top_n=False): 
+    """
+    Summarizes results of `_grid_randomization` into a matrix where each row is a gene 
+    and each column is a parameter run. If summarizing "membership", entries will be 0 or 1
+    indicating whether or not a node appeared in each experiment. If summarizing "robustness"
+    or "specificity", entries indicate robustness or specificity values for each experiment. 
+    Also, node attributes columns are added for plotting purposes.
```

Comment:
> Also, node attributes columns are added for plotting purposes.

Can you expand on this?

Reply:
7e60508ed6ec7e22e3340a7608f3056dddc61544

Thought I removed that line. Node attributes are kept as a separate dataframe.